strategies/categorized/pom.xml

```diff
@@ -31,6 +31,13 @@
       <artifactId>assertj-core</artifactId>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.arquillian.smart.testing</groupId>
+      <artifactId>core</artifactId>
+      <version>${project.version}</version>
+      <classifier>tests</classifier>
+      <scope>test</scope>
+    </dependency>
```

Comment:
Can you move this in `dependencyManagement` of parent pom.xml and use it like 
```xml
<dependency>
      <groupId>org.arquillian.smart.testing</groupId>
      <artifactId>core</artifactId>
      <classifier>tests</classifier>
      <scope>test</scope>
    </dependency>
```

Reply:
I'm not really a fan of having everything inside of the parent pom file - it's getting to be bigger and bigger. 
In addition, in the parent pom file, it is a part of the `dependencyManagement` section which takes care of the versions - to use the correct one. But in this case, there is nothing to manage - it is `${project.version}`. So putting 6 lines with another dependency inside of the parent pom file (which is quite long already) just because of saving 2 lines in these pom files (that is really not necessary) doesn't seem to be worth doing for me. Just my $0.02. WDYT @lordofthejars ?

Reply:
@MatousJobanek I agree with you, but if you check other Arquillian projects (some of them not all) we are doing how @dipak-pawar mentions. For me both approaches are ok, what I would say is let's see how we are using in Smart Testing and apply the same logic.

Reply:
> what I would say is let's see how we are using in Smart Testing and apply the same logic.

And this is exactly what I don't like and I wanted to start making more clear (mainly in this case of the jar with testing bits). Anyway, I'll move it there and create another issue to remove those dependencies from the parent pom file that don't make sense to have them there.